Sources/Public/Animation/LottieAnimationView.swift

```diff
@@ -103,18 +26,15 @@ open class LottieAnimationView: LottieAnimationViewBase {
     configuration: LottieConfiguration = .shared,
     logger: LottieLogger = .shared)
   {
-    self.animation = animation
-    self.imageProvider = imageProvider ?? BundleImageProvider(bundle: Bundle.main, searchPath: nil)
-    self.textProvider = textProvider
-    self.fontProvider = fontProvider
-    self.configuration = configuration
+    self.lottieAnimationLayer = LottieAnimationLayer(animation: animation, imageProvider: imageProvider, textProvider: textProvider, fontProvider: fontProvider, configuration: configuration, logger: logger)
     self.logger = logger
     super.init(frame: .zero)
     commonInit()
-    makeAnimationLayer(usingEngine: configuration.renderingEngine)
     if let animation = animation {
       frame = animation.bounds
     }
+    self.viewLayer?.addSublayer(self.lottieAnimationLayer)
+    setNeedsLayout()
```

Comment:
Should these two items go in `commonInit`? It seems like we do this in every `init`